app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
If you're removing it from the API, and recounting every time you upload a subject, why have the counter in the DB at all?